mne/minimum_norm/inverse.py

```diff
@@ -331,6 +331,8 @@ def write_inverse_operator(fname, inv, verbose=None):
     """
     check_fname(fname, 'inverse operator', ('-inv.fif', '-inv.fif.gz',
                                             '_inv.fif', '_inv.fif.gz'))
```

Comment:
Might be better / simpler just to make `check_fname` call `_check_fname` internally and return the `str`-ified absolute path, then our code ends up more DRY. It seems like every time we `check_fname` we probably want to call `_check_fname` anyway...

Or we could go the other way and make `_check_fname` accept a `ext` argument, and have it call `check_fname` internally. I think I'd rather go that way.

Reply:
Yes I was thinking the same, however I don't want to touch this in this PR to avoid unwelcome surprises and to keep the diff smaller.